make/run-cicd-script.mk

```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=xcoulon/toolchain-cicd/d426c7b
```

Comment:
and this one too

Reply:
reverted in https://github.com/codeready-toolchain/toolchain-e2e/commit/7e01e2fe1f9159f8a27001e50f1c5e881390ad10